DigitalOffice/Services/CheckRightsService/Repositories/CheckRightsRepository.cs

```diff
@@ -19,5 +21,30 @@ public List<DbRight> GetRightsList()
         {
             return dbContext.Rights.ToList();
         }
+
+        public bool AddRightsToUser(RightsForUserRequest request)
+        {
+            foreach (var rightId in request.RightsId)
+            {
+                var dbRight = dbContext.Rights.FirstOrDefault(right => right.Id == rightId);
+                if (dbRight == null)
+                {
+                    //TODO change on custom exception
+                    throw new Exception("Right doesn't exist.");
+                }
```

Comment:
NullReferenceException

Reply:
Do you mean change on NullReferenceException?